content/management/configuration-api/v3.4/index.mdx

```diff
@@ -4625,12 +4626,12 @@ Informs that a bot was deleted.
 
 ### `group_created`
 
-Informs that a group has been created within a license.
+Informs that a group has been created within a license. Response will contain full state of group resource - empty fields won't be omitted.
```

Comment:
The same pattern as above with link to `/management/configuration-api/v3.4/#groups`